projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -74,21 +113,44 @@ export function NoticeBox( props ) {
 				),
 				props.planRecordLimit
 			),
+			link: {
+				text: __( 'learn more', 'jetpack-search-pkg' ),
+				url: 'https://jetpack.com/support/search/product-pricing/',
+			},
 		} );
 	}
 
-	if ( ! notices || notices.length < 1 ) {
+	//remove any dismissed notices from notices array
+	for ( const notice in notices ) {
+		const noticeID = notices[ notice ].id;
+
+		if ( dismissedNoticesArray.includes( noticeID.toString() ) ) {
+			notices.splice( notice );
+		}
+	}
+
+	if ( ! notices || notices.length < 1 || ! showNotice ) {
 		return null;
 	}
 
 	const noticeBoxClassName = notices[ 0 ].isImportant
-		? 'jp-search-notice-box__important'
+		? 'jp-search-notice-box jp-search-notice-box__important'
 		: 'jp-search-notice-box';
 
 	return (
-		<div data-testid="notice-box" className={ noticeBoxClassName }>
-			<p>{ notices[ 0 ].message }</p>
-		</div>
+		<SimpleNotice
+			isCompact={ false }
+			status={ 'is-info' }
+			className={ noticeBoxClassName }
+			onDismissClick={ dismissNoticeBox }
+		>
+			{ notices[ 0 ].message }
```

Comment:
Nice to have:

If we are going to display on one notice, should we have some kind of priorities? and sort the notice array before picking the first one to show?

Reply:
Yes, I've tried to implement this already by prioritising the order in which notices get added to the list, so that whichever is at item [0] should be the highest priority item. But I do agree some kind of sort by priority could be a nice approach too. I shall have a think on this one!